### DISCUSSION TITLE
Certain files are crashing my HD300's like crazy.  Thoughts?

### DISCUSSION THREAD

**Discussion by Sagetvmaster** [OpenSageTV/Community#143] [SageTV Extenders (HD300,HD200)]
Lately, files that may be using e-AC3 audio encodes, or certain .mkv codecs have been crashing my HD300's like crazy.  I'm either met with insanely loud digital distortion, or the unit just locks up.  I know it's been 14-years since the hardware was last manufactured, but are there any workaround for more recent codecs and audio formats, or do we have to convert everything to a compliant file.
Is there a master-list somewhere which outlines all usable audio/video codecs, etc.?

**Comment by pendragonsound** [comment]
The HD300 manual specifies the audio formats as "MP2, MP3, AAC, AAC-HE, ALAC, WMA, WMAPro, PCM, Vorbis (stereo only), FLAC, DolbyDigital/DolbyDigital+/DolbyTrueHD* (stereo down-mix or pass-through), DTS/DTS-HD/DTS-MA (stereo down-mix or pass-through)". In terms of EAC3 and others, this may depend on the AC3 core track being present, which I am not sure is required by EAC3. The success the HD300 has may depend on the capabilities of the downstream AVR/TV/pre-pro and what is negotiated via HMDI. These negotiations are delicate and sometimes inconsistent. If the HD300 can negotiate a bitstream passthrough, I would expect it would be much less vulnerable to the actual format of the stream. I've had mixed luck with MKV files and the HD300. I've produced original and high-end AV content and mostly succeeded in finding codec combinations that work with the HD300. I'd have to go back through my notes to have anything more to offer. Do you have control of the format of these renegade files, or do they come from other sources/software?

**Comment by wayner9** [comment]
I would think that the best solution is to re-encode with something like Handbrake.

**Reply by Sagetvmaster** [reply]
turns out it was the audio selection in setup - bitstream vs PCM.  PCM was selected for some reason.  AI takeover.   lol
